terraso_backend/apps/graphql/schema/schema.graphql

```diff
@@ -1958,6 +1959,7 @@ input SiteAddMutationInput {
   name: String!
   latitude: Float!
   longitude: Float!
+  elevation: Float!
```

Comment:
comment: i don't think it's a blocker, but we should file a follow-up tech-debt issue to maintain awareness of it, it seems like there's a bit of shtuff to think through re whether the client will always be able to supply an elevation. like if our elevation API provider goes down no new sites can be created, or if a user creates a site offline we need to insert a step where we compute the elevation when we get back online. but again i think this is ok for now